data-prepper-plugins/s3-source/README.md

```diff
@@ -160,6 +160,8 @@ The AWS configuration is the same for both SQS and S3.
 
 * `region` (Optional) : The AWS region to use for credentials. Defaults to [standard SDK behavior to determine the region](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/region-selection.html).
 * `sts_role_arn` (Optional) : The AWS STS role to assume for requests to SQS and S3. Defaults to null, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html). 
+* `sts_external_id` (Optional) : The external ID to attach 
```

Comment:
Update the documentation here to be consistent with other docs.
The external ID to attach to AssumeRole requests.

Reply:
Thanks for catching that. Fixed.